### PR TITLE
Feature add options

### PR DESCRIPTION
**Pull request by lucasfth**
Click `h` to hide and unhide the menu

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -105,6 +210,10 @@ $(document).on('keypress', function (e) {
   if (e.which == 13) {
     spin(2);
   }
+  // listen on h key and hide and unhide
+  if (e.which == 104) {
+    menuItem.hidden = !menuItem.hidden;
+  }
```
Comment (A-Guldborg) [inline script.js]: Can we make it hidden by default? This way, we don't expose it to customers when turning on the wheel

File: script.js [hunk 2]
```diff
@@ -24,6 +14,121 @@ var data = [
...
+    .attr('stroke', 'black')
+    .attr('font-align', 'center')
+    .attr('stroke-width', 0.6)
+    .attr('fill', function(d,i){
+      return (i % 2 == 1) ? "black" : "white";
+    })
+    .attr('text-anchor', 'end')
+    .text(function (d, i) {
+      return data[i].label;
+    });
```
Comment (A-Guldborg) [inline script.js]: This looks to me to be duplicate of existing code further down in the file used to render the wheel itself with new options. Can we make it into a separate function and call the same function both places to ensure consistency?

File: script.js [hunk 3]
```diff
@@ -24,6 +14,121 @@ var data = [
...
+// Init data filled in the wheel
+var data = og;
+
+// Different options for the wheel
+var options = {
+  "og": og,
+  "mixers": mixers,
+  "shots": shots,
+  "sirup": sirup
+};
```
Comment (A-Guldborg) [inline script.js]: Would be very cool if these options were easily editable "semi permanently".
The "easy" solution without a database would probably be to define `og.txt` and then have an interface to write to and load from this file through the website. Doesn't need to be git-controlled but could be so for the first few options at least. But one might consider this a new feature.

**Review by A-Guldborg — CHANGES_REQUESTED** [review]

File: script.js (hunk 3 above)
Comment (lucasfth) [reply script.js]: Resolved

File: script.js (hunk 2 above)
Comment (lucasfth) [reply script.js]: Resolved

File: script.js (hunk 1 above)
Comment (lucasfth) [reply script.js]: Resolved

File: style.css [hunk 4]
```diff
@@ -168,3 +168,13 @@ body {
...
 }
+
+#wheelInput {
+  position: fixed;
+  bottom: 20px;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 50%;
+  margin: 0 auto;
+  z-index: 1000;h
```
Comment (A-Guldborg) [inline style.css]:
```suggestion
  z-index: 1000;
```

File: script.js [hunk 5]
```diff
@@ -1,4 +1,71 @@
...
+
+fileUpdate.addEventListener('change', function (e) {
+  const file = e.target.files[0];
+  if (file) {
+    const reader = new FileReader();
+    reader.onload = function (e) {
+      const tmp = e.target.result.split('\n');
+      data = tmp.map((item) => {
+        return { label: item.trim() };
+      }).filter((item) => item.label.length > 0);
```
Comment (A-Guldborg) [inline script.js:26]: Should we add a validation here to ensure that the input matches our expected format, i.e. 12 items one per line?
Then send an alert("malformed") if not.

File: script.js [hunk 6]
```diff
@@ -105,6 +123,10 @@ $(document).on('keypress', function (e) {
   if (e.which == 13) {
     spin(2);
   }
+  // listen on h key and hide and unhide
+  if (e.which == 104) {
```
Comment (A-Guldborg) [inline script.js]: Would it work as the following? And possibly also on enter '\r' possibly?

```suggestion
  if (e.which == 'h') {
```

File: script.js (hunk 5 above)
Comment (lucasfth) [reply script.js:26]: I have added check

File: script.js (hunk 6 above)
Comment (lucasfth) [reply script.js]: Done

**Review by A-Guldborg — APPROVED** [review]

**Comment by lucasfth** [comment]
Yush

**Pull request merged**